contents/docs/data/persons.mdx

```diff
@@ -60,9 +63,21 @@ client.capture({
 })
 ```
 
```

Comment:
Should we have a link out to the user properties page here saying "Here's the docs on how to set user properties on person profiles"?